sympy/combinatorics/coset_table.py

```diff
@@ -766,12 +766,15 @@ def conjugates(self, R):
             R_set.difference_update(r)
         return R_c_list
 
-    def coset_representative(self, alpha):
+    def coset_representative(self, coset):
         '''
-        Compute the coset representative of a coset alpha
+        Compute the coset representative of a given coset.
         '''
-        # To be implemented
-        return None
+        for alpha, x in product(self.omega, self.A):
+            if self.table[alpha][self.A_dict[x]] == coset:
+                if alpha == 0:
+                    return x
+                return self.coset_representative(alpha)*x
```

Comment:
It'd be much more efficient to start in `self.table[coset]`, find the first smaller entry `alpha = self.table[coset][x]` and then `return x**-1*(self.coset_represenative(alpha))**-1`. But in any case, I think coset representatives for schreier generators should be computed differently.